.github/workflows/checkov.yml

```diff
@@ -0,0 +1,35 @@
+name: Run checkov
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+    - v2m*
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
```

Comment:
Minor: Do we need to set up a python version? If so can we setup python3.9 which is what we use elsewhere for data.all?

Reply:
I have updated the Python version on all actions now. I have also renamed them to be more descriptive. Static checking was doing the same as flake8 so I have deleted it